utils/helpers/recommendations/index.ts

```diff
@@ -0,0 +1,81 @@
+import { IMumbleUser } from '@/utils/interfaces/mumbleUsers.interface';
+
+export const completeRecommendations = ({
+  maxAmount,
+  rawUsers,
+  followedUsersIdentifiers,
+  rejectedUsersIdentifiers,
+  currentRecommendations,
+}: {
+  maxAmount: number;
+  rawUsers: IMumbleUser[];
+  currentRecommendations: IMumbleUser[];
+  followedUsersIdentifiers: string[];
+  rejectedUsersIdentifiers: string[];
+}): IMumbleUser[] => {
+  const selectableUsers = getSelectableUsers({
+    rawUsers,
+    followedUsersIdentifiers,
+    rejectedUsersIdentifiers,
+  });
+  const currentAmount = currentRecommendations.length;
+  const neededAmount = maxAmount - currentAmount;
+  if (selectableUsers.length === 0) {
```

Comment:
what do you think of using two if-statements (guard clauses) instead of an else-if statement? this would make it more readable. 
```
if (selectableUsers.length === 0) {
    return [...currentRecommendations];
  } 
 if (selectableUsers.length <= neededAmount) {
    return [...currentRecommendations, ...selectableUsers];
  }
```

Reply:
I will keep it with the else part. 
More understandable for me is to have the else as only one conditions should apply.